specialScenarios/scenarioBuyFerry.py

```diff
@@ -0,0 +1,27 @@
+import random
```

Comment:
This logic should be incorporated in the River scenario. It should not be its own scenario